cmd/vsphere-xcopy-volume-populator/internal/populator/populate_test.go

```diff
@@ -0,0 +1,110 @@
+package populator_test
+
+import (
+	"testing"
+
+	"github.com/kubev2v/forklift/cmd/vsphere-xcopy-volume-populator/internal/populator"
+)
+
+func TestVMDisk_Path(t *testing.T) {
```

Comment:
We already fixed the parsing in the main branch, and took a bit different approach, would be cool if we could use the tests, though they need to be adapted a little.